DFC.App.ExploreCareers.Data/Contracts/IApiService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace DFC.App.ExploreCareers.Data.Contracts
+{
+    public interface IApiService
```

Comment:
You do not  need to define this interface yourself as it is defined in the content nuget